wgs/wasm/deliverables/wasm-oci-artifact.md

```diff
@@ -0,0 +1,132 @@
+        {
+            "mediaType": "application/wasm",
+            "size": 692031,
+            "digest": "sha256:1234567890"
+        },
+    ]
+}
+```
+
+## Config.mediaType (`application/vnd.wasm.config.v0+json`)
+
+The configuration provides the ability to quickly identify imports, exports or worlds that are used by the component. By specifying imports and exports, 
+it ensures a runtime can validate that the component has all of the layers required to satisfy the exports. It also ensures that a Wasm runtime can reject 
+running the a component if it can not satisfy the imports.
+
+```
+{
+    "created": "2015-10-31T22:22:56.015925234Z",
+    // This can be set here or as an annotation
+    "author": "Alyssa P. Hacker <alyspdev@example.com>",
+    "architecture": "wasm",
+    // Eventually this will go away when we hit a 1.0 but we need it for now
+    // Possible options: wasip1, wasip2. For plain wasm, this should be wasip1 as this must match a GOOS value and it doesn’t have one for plain Wasm
+    "os": "wasip2",
+    // We need to have a unique list here so that the hash of the config (used as the ID) is unique every time (https://github.com/opencontainers/image-spec/pull/1173). These are just the digests of the layers in the same order
+    "layerDigests": [
+        "sha256:123456",
+        "sha256:abcdef"
+    ],
+    // This field MUST be present when it is a component (wasip2)
+    "component": {
+        // Decision: Optional imports will be a thing in the future, so when that happens we will add an additional field called "optional_imports"
+        "exports": [
+            "wasi:http/incoming-handler@0.2.0",
+            "function-name"
+        ],
+        "imports": [
+            "wasi:filesystem/types@0.2.0",
+            "function-name"
+        ],
+        // This is optional metadata for indexing. Implementations MAY use this information to fetch other data to inspect the specified world
+        "target": "wasi:http/proxy@0.2.0",
+    },
+    // In the future, we could add an optional "module" section that is only present when the OS value is wasip1. This could carry additional config data or other options. 
+    // However, we are not adding this to start but would be more than willing to add it in as concrete use cases emerge. For components, config and other data will be
+    // handled via composition using tools like wasi-virt
+}
+```
+
+## FAQ
+
+#### If there is a single layer type how will static files or runtime configuration files be supplied?
+
+- Runtime configuration: WASI spec like  https://github.com/WebAssembly/wasi-runtime-config/pull/4 or specialized component that the host runtime would require 
+- Static files: Virtual file system - https://github.com/bytecodealliance/wasi-virt?tab=readme-ov-file#filesystem or specialized component that could be provided to do further customization
+
+## Implementations
+No current implementations but the following projects have interest:
+
+- runwasi/containerd
```

Comment:
Nit: Capitalize project names?

Reply:
`containerd` is lowercase, so maybe runwasi is too? We have it lower case in our readmes🤷